docs/conf.py

```diff
@@ -22,13 +24,16 @@
 # -- Project information -----------------------------------------------------
 
 project = "stdpopsim"
-copyright = "2019-2022, PopSim Consortium"
+copyright = f"2019-{datetime.datetime.now().year}, PopSim Consortium"
 author = "PopSim Consortium"
 
-# The short X.Y version
-version = ""
-# The full version, including alpha/beta/rc tags
-release = "0.2.0"
+# get the release version from the setuptools_scm
+try:
+    release = get_version(root="..", relative_to=__file__)
+    version = ".".join(release.split(".")[:2])
```

Comment:
What's the `[:2]` intended to chop out here? (Note that I can't get the version number to display anywhere in a local docs build). But, if I run this via the REPL, I get that `release == '0.2.1a2.dev65+g2d5ba65'` and `version == 0.2`. 

I don't think we want to chop off this much, if anything? That is, stable builds should have a nice looking version number (so no need to chop anything off). Unstable builds will have the commit hash, which looks ugly but is providing necessary information IMO (so also don't want to chop that off).

Reply:
yeah that's what i was intending-- that version would be `version == 0.2` and release would be that blob. I've pushed a commit that manually adds a footer with all this info displayed. also I *think* once we merge the version selector business will appear on RTD